frontend/src/views/Login/Mfa.tsx

```diff
@@ -110,14 +112,19 @@ export const Mfa = ({ successCallback, closeMfa, hideLogo, email, method }: Prop
 
   if (shouldShowTotpRegistration) {
     return (
-      <div className="mx-auto w-max pb-4 pt-4 md:mb-16 md:px-8">
-        <TotpRegistration
-          onComplete={async () => {
-            setShouldShowTotpRegistration(false);
-            await successCallback();
-          }}
-        />
-      </div>
+      <>
+        <div className="mb-6 text-center text-lg font-bold text-white">
+          Your organization requires mobile authenticator to be configured.
```

Comment:
nit: `Your organization requires a mobile authenticator to be configured.` or `Your organization requires mobile authentication to be configured.`